### DISCUSSION TITLE
Welcome to MONAILabel Discussions!

### DISCUSSION THREAD

**Discussion by SachidanandAlle** [Project-MONAI/MONAILabel#79] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by curtislisle** [comment]
Thanks to the authors, for working on MONAILabel.  I think this is a promising tool for AI-assisted segmentation.   I am an imaging and data science consultant working on deep learning and visualizaiton applications for both radiology and pathology images.   After building and using MONAILabel, I like it.  I have a few suggestions for different applications that I will list in a new discussion thread.  

Please keep up this good work.  I like this application.
